src/7-to-8/cheat-sheet.rst

```diff
@@ -260,6 +260,29 @@ Generate a visualisation for a workflow without running it:
 
        The web UI will have full graph vis. in a future release.
 
+Datetime Operations
+-------------------
+
+Datetime operations in task scripts:
+
+.. list-table::
+   :class: grid-table
+
+   * - **Cylc 7** & Rose 2019
+     - **Cylc 8** (Rose 2)
+   * - ::
+
+         rose date -c
+     - ::
+
+         isodatetime ref
```

Comment:
Just a comment : I like the syntax synopsis as you have defined it in the linter with the square brackets as this makes it clearer that the `ref` is replacing the `-c`, i.e.

`rose date [-c]` => `isodatetime [ref]`